echopype/convert/set_groups_ek80.py

```diff
@@ -165,9 +165,9 @@ def set_platform(self) -> xr.Dataset:
                     },
                 ),
                 "sentence_type": (["location_time"], msg_type),
-                "water_level": (
+                "range_offset": (
                     [],
-                    water_level,
+                    range_offset,
                     {
                         "long_name": "z-axis distance from the platform coordinate system "
```

Comment:
In generalizing from `water_level` to `range_offset`, the long_name description "z-axis distance ..." doesn't seem to be strictly accurate any more. It seems to me that it should be something like "Along-range distance ..."

Reply:
So, I just went back to check the [convention](https://www.ices.dk/sites/pub/Publication%20Reports/Cooperative%20Research%20Report%20(CRR)/CRR341.pdf) and there is actually no `range_offset` in there but there is `water_level`. 😭   But there is also `vertical_offset`...  🤯   I am now very confused, both in how we (was that me?) arrived at changing things to `range_offset` (although I do think this is more general) and the relationship between the two.